teams/delius-iaps/components/iaps_server/delius_iaps_install_base_packages.yml

```diff
@@ -27,6 +27,8 @@ phases:
               Write-Host('vcredist packages')
               # choco install -y vcredist2010 # Commented out - hoping we don't need to explicitly mention this due to dynamic package dependency
               # choco install -y vcredist2008 # Commented out - hoping we don't need to explicitly mention this due to dynamic package dependency
+              cinst -y vcredist140
```

Comment:
`cinst` appears to be an alias of `choco install`. is that right? can we use one version of the command in this file for consistency?